.eslintrc.js

```diff
@@ -0,0 +1,199 @@
+module.exports = {
+  root: true,
+  overrides: [
+    {
+      /*
+       * REACT
+       */
+      files: ['**/*.js'],
+      env: {
+        es2020: true,
+        commonjs: true, // Needed to avoid import is reserved error
+        node: true,
+        browser: true,
+        jest: true,
+      },
+      parser: 'babel-eslint',
+      extends: [
+        'eslint:recommended',
+        'plugin:react/recommended',
+        'plugin:prettier/recommended',
+      ],
+      plugins: ['react', 'import'],
```

Comment:
What does the `import` plugin do ? 
In the list of plugins I usually have `prettier` also, should we add it here?